tools/cdf_creator/swe_cdf_creator.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""
+swe_cdf_creator.py
+usage: <python> <swe_cdf_creator.py> <configuration.cfg>
+
+Creates list of ascending list from long-term SWE VIC produced data.
```

Comment:
"Creates ascending list from simulated long-term SWE" instead? This can be used for other model output, right?